src/libraries/System.Net.Security/tests/FunctionalTests/CertificateValidationRemoteServer.cs

```diff
@@ -101,12 +101,17 @@ public Task ConnectWithRevocation_WithCallback(bool checkRevocation)
         }
 
         [PlatformSpecific(TestPlatforms.Linux)]
-        [Theory]
+        [ConditionalTheory]
         [OuterLoop("Subject to system load race conditions")]
         [InlineData(false)]
         [InlineData(true)]
         public Task ConnectWithRevocation_StapledOcsp(bool offlineContext)
         {
+            if (PlatformDetection.IsRedHatFamily7)
```

Comment:
The issue says it's only failing when offlineContext is false.  Should this only skip when it's false?

Reply:
Seems reasonable.